File: php/facadePattern/Passenger.php
<?php  namespace php\facadePattern;
/*
|==========================================================================
|--------------------------------------------------------------------------
|	The values for the properties would usually be obtained from a database
| 	Values have been explicitly added for example purposes.
|--------------------------------------------------------------------------
|==========================================================================
*/
class Passenger{
	
       /**
	* Name of the passenger
	* 
	* @var string $passengerName
	*/
	private $passengerName = 'Bob Brown'; 
	
       /**
	* Address of the passenger
	* 
	* @var string $passengerAddress
	*/
	private $passengerAddress = 'London';
	
       /**
	* Age of the passenger
	* 
	* @var string $passengerAge
	*/
	private $passengerAge = '41';
	
       /**
	* The passenger ID
	* 
	* @var string $id
	*/
	private $id;

	public function __construct($passID){
		$this->id = $passID;
	}

       /**
	* Get the passenger name
	* 
	* @return string
	*/
	public function getPassengerName(){
		
		return $this->passengerName;
	}
	
       /**
	* Get the address of the passenger
	* 
	* @return string
	*/
	public function getPassengerAddress(){
		
		return $this->passengerAddress;
	}
	
       /**
	* Get the age of the passenger
	* 
	* @return string
	*/
	public function getPassengerAge(){
		
		return $this->passengerAge;
	}
}

File: java/facadePattern/Passenger.java
package facadePattern;
/*
|========================================================================
|------------------------------------------------------------------------
| The values for the instance variables would 
|  usually be obtained from a database
|  However values have been added for the benfit of this example
|------------------------------------------------------------------------
|========================================================================
*/
public class Passenger {
	private String id;
	private String passengerName = "Billy Bob";
	private String passengerAddress = "New York";
	private String passengerAge = "23";

	public Passenger(String id) {
		this.id = id;
	}

	public String getPassengerName() {
		return this.passengerName;
	}

	public String getPassengerAddress() {
		return this.passengerAddress;
	}

	public String getPassengerAge() {
		return this.passengerAge;
	}
}
